the-collab-lab/tcl-10-smart-shopping-list
# Remove unused styling

## Description

- Remove *.module.css files. 
- Remove unused styles from App.css. 
- Remove inline styling from List.js and put styling in App.css. 
- Delete header tag from App.js. 
- Add simple styling to keep List.js ul list in place.

Why did I choose this approach?

In order to prepare for creating new Material-UI components with styling I thought it would be important to remove all leftover and unused styling. This will prevent old unused styles from conflicting with new styles.

## Related Issue

Closes #56 

## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|    | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |
|  ✓ | :sparkles: Styling update     |

## Updates

### Before

#### AddItem.js

<img width="313" alt="Screen Shot 2020-09-23 at 12 33 00 PM" src="https://user-images.githubusercontent.com/25488451/94041958-ee1dad00-fd98-11ea-8f6e-92957b498097.png">

#### Details.js 

<img width="314" alt="Screen Shot 2020-09-23 at 12 32 41 PM" src="https://user-images.githubusercontent.com/25488451/94041923-e231eb00-fd98-11ea-84b7-59170aeb83f1.png">

#### List.js

<img width="312" alt="Screen Shot 2020-09-23 at 12 32 22 PM" src="https://user-images.githubusercontent.com/25488451/94041891-d5ad9280-fd98-11ea-9c02-381326cc4625.png">

#### Welcome.js

<img width="313" alt="Screen Shot 2020-09-23 at 12 32 00 PM" src="https://user-images.githubusercontent.com/25488451/94041854-c9c1d080-fd98-11ea-923a-250772a4b6bf.png">


### After

#### AddItem.js

<img width="318" alt="Screen Shot 2020-09-23 at 12 28 01 PM" src="https://user-images.githubusercontent.com/25488451/94041472-430cf380-fd98-11ea-8b2b-1689790ade42.png">

#### Details.js 

<img width="316" alt="Screen Shot 2020-09-23 at 12 28 38 PM" src="https://user-images.githubusercontent.com/25488451/94041512-5324d300-fd98-11ea-8f83-debe3437a474.png">

#### List.js

<img width="314" alt="Screen Shot 2020-09-23 at 12 26 52 PM" src="https://user-images.githubusercontent.com/25488451/94041309-12c55500-fd98-11ea-8c87-3343fe19daa4.png">

#### Welcome.js

<img width="316" alt="Screen Shot 2020-09-23 at 12 30 02 PM" src="https://user-images.githubusercontent.com/25488451/94041635-836c7180-fd98-11ea-9ff5-d3b5696d13e9.png">


## Testing Steps / QA Criteria

- Check to make sure the App behaves no differently than before and no useful styling is removed.


- Ignore styling from components that are not yet completed and are still being built using Material-UI
